spherogram_src/codecs/DT.py

```diff
@@ -289,7 +289,7 @@ def marked_arc(self, vertex):
                 if V == vertex:
                     raise ValueError('Marked graph is a circle')
                 edges = [e for e in self(V) if e.marked and e != edge]
-                if len(edges) == 0:
+                if not edges:
```

Comment:
I prefer the original form of `len(edges) == 0`, as `not edges` makes me think of negating a list, which is nonsensical.  Some with other instances of this in this PR.